src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorWorkflowStep.groovy

```diff
@@ -120,6 +128,11 @@ class IqPolicyEvaluatorWorkflowStep
       IqUtil.doFillIqStageItems(jobCredentialsId, job)
     }
 
+    @Override
+    FormValidation doCheckAdvancedProperties(@QueryParameter String advancedProperties) {
+      // TODO validate that the properties are entered into the correct format?
```

Comment:
Same as the other `TODO` comment above. Maybe use the same ticket for these if you create one.